R/FilterStateChoices.R

```diff
@@ -1,16 +1,24 @@
+# ChoicesFilterState ------
+
 #' @name ChoicesFilterState
-#' @title `FilterState` object for factor or character variable
-#' @description Manages choosing elements from a set
 #' @docType class
-#' @keywords internal
 #'
+#' @title `FilterState` object for factor or character variable
+#'
+#' @description Manages choosing elements from a set.
 #'
 #' @examples
-#' filter_state <- teal.slice:::ChoicesFilterState$new(
+#' # use non-exported function from teal.slice
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
+#' include_css_files <- getFromNamespace("include_css_files", "teal.slice")
+#' include_js_files <- getFromNamespace("include_js_files", "teal.slice")
```

Comment:
@kartikeyakirar why include_js_files <- getFromNamespace("include_js_files", "teal.slice") is repeated twice?